app.py

```diff
@@ -97,7 +95,7 @@ def login():
         else:
             error = 'Username not found'
             return render_template('login.html', error=error)
-
+        return name
```

Comment:
Umm?

Reply:
leftover from attempt to get username out to put in last_updated_by...